Rubberduck.Parsing/ComReflection/ComModule.cs

```diff
@@ -23,6 +24,8 @@ public class ComModule : ComType, IComTypeWithMembers, IComTypeWithFields
         private readonly List<ComField> _fields = new List<ComField>();
         public IEnumerable<ComField> Fields => _fields;
 
+        public IEnumerable<ComField> Properties => Enumerable.Empty<ComField>();
```

Comment:
`Properties` is read-only. The inheritance and interface structure in this namespace could probably use some work eventually, but in this case I put it on the `IComTypeWithMembers` interface because at this point I don't have any evidence that these are prohibited on Modules other than never having seen any. If I can conclusively determine from the MIDL spec that they'll never be there, I'll probably rearrange this.